dpdd.tex

```diff
@@ -1068,6 +1091,82 @@ \subsubsection{\ForcedSource Table}
 flags & bit[8] & bit & Various useful flags. \\
 \end{schema}
 
+\subsubsection{\texttt{ShearObject} Table}
+\label{sec:shearObjectTable}
+
+% ShearObject table is a conventions-translation and generalization of a schema
+% of the DES metadetect catalog schema from Matt Becker; differences include:
+%
+% x,y: dropped as redundant with ra/dec
+%
+% gauss_*: prefixes dropped as irrelevant in this context.
+%
+% gauss_T_ratio: dropped as redundant with T/TPSFMeta.  We might want to
+% restore it in the physical schema so we can index it separately.
+%
+% pgauss_*: prefixes replaced with "std[Flux]" to emphasize conceptual
+% relationship with Object's "stdColors".
+%
+% PSF moment measurements: I've assumed full (g1, g2, T) measurements on both
+% the metacalibration-modified PSFs and the originals, but dropped pgauss_psf_T
+% as too algorithmic-specific for the logical schema.
+%
+% I've expanded the pgauss_T field and its uncertainty into per-band
+% measurements, as the apparent cross-band average in the original schema also
+% seems too algorithm-specific, and I'd like to err on the side of predicting a
+% larger table than we might need.
+
+\begin{schema}{\texttt{ShearObject} Table}{Data Release Catalog \texttt{ShearObject} Table}{tbl:shearObjectTable}
+
+shearObjectId & uint64 & ~ & Unique object identifier. \\
+
+cellId & uint64 & ~ & Unique identifier for the coadd cell in which this object was detected and measured. \\
+
+metaStep & int8 & ~ & Enumeration value for which of the five \texttt{Metacalibration} modifications was applied to images prior to all measurements. \\
+
+radec & double[2] & degrees & Position of the object (centroid), measured jointly across all bands. \\
```

Comment:
No radecErr?

Reply:
FWIW, we don't record this. For static science, there is no real use for this kind of error. All of the scales we care about are orders of magnitude above the scale of our centroiding errors.